### PR TITLE
docs/node: Add info box for Emerald version for Eden upgrade

### PR DESCRIPTION
**Pull request by amela**

### CODE REVIEW

File: docs/node/mainnet/eden-upgrade.md [hunk 1]
```diff
@@ -273,6 +273,13 @@ network, while the other is used to bootstrap the Oasis P2P network.
 
 :::
 
+:::danger
```
Comment (ptrus) [inline docs/node/mainnet/eden-upgrade.md]: Could maybe also add it  as step 11 in the upgrade procedure (similar as for rosseta gateway)

>11 (only Emerald paratime operators) Upgrade Emerald to Version 11.0.0.

Comment (amela) [reply docs/node/mainnet/eden-upgrade.md]: Added

Comment (kostko) [reply docs/node/mainnet/eden-upgrade.md]: Also no need to mark it as "danger", it can be "info".

Comment (amela) [reply docs/node/mainnet/eden-upgrade.md]: Done

File: docs/node/mainnet/eden-upgrade.md [hunk 2]
```diff
@@ -273,6 +275,13 @@ network, while the other is used to bootstrap the Oasis P2P network.
 
 :::
 
+:::info
+
+For those running Emerald Paratime, please make sure to upgrade to Emerald
+Version 11.0.0 before or soon after the network upgrade is completed.
```
Comment (kostko) [inline docs/node/mainnet/eden-upgrade.md]:
```suggestion
For those running the Emerald ParaTime, please make sure to upgrade to Emerald
version 11.0.0 before or soon after the network upgrade is completed.
```

File: docs/node/mainnet/eden-upgrade.md [hunk 3]
```diff
@@ -203,7 +203,9 @@ Ubuntu versions (or other distributions using older system libraries).
 10. (only Rosetta Gateway operators) Replace old version of Oasis Rosetta
     Gateway with version [2.6.0][rosetta-gw-2.6.0].
 
-11. Start your node and re-enable auto-starting via your process manager.
+11. (only Emerald paratime operators) Upgrade Emerald to Version 11.0.0.
```
Comment (kostko) [inline docs/node/mainnet/eden-upgrade.md]:
```suggestion
11. (only Emerald paratime operators) Upgrade Emerald to version 11.0.0.
```

**Review by kostko — APPROVED** [review]

**Pull request merged**